pkgs/samples/lib/google_ai.dart

```diff
@@ -157,19 +159,40 @@ class _ChatWidgetState extends State<ChatWidget> {
 
   @override
   Widget build(BuildContext context) {
-    final history = _chat.history.toList();
+    var textFieldDecoration = InputDecoration(
```

Comment:
Is this the same as the `textFieldDecoration` method you added below to share with the API screen? Do you want to keep sharing that implementation or nah?